bindings/Sofa/src/SofaPython3/Sofa/Simulation/Submodule_Simulation.cpp

```diff
@@ -67,21 +69,24 @@ PYBIND11_MODULE(Simulation, simulation)
 
 
     simulation.def("updateVisual", [](Node* n){ sofa::simulation::getSimulation()->updateVisual(n); });
+
+#ifndef SOFA_NO_OPENGL
     simulation.def("draw", [](Node* n){
         auto* vparam = sofa::core::visual::VisualParams::defaultInstance();
         vparam->drawTool() = new sofa::core::visual::DrawToolGL();
         vparam->setSupported(sofa::core::visual::API_OpenGL);
         sofa::simulation::getSimulation()->draw(vparam, n);
     });
 
-    simulation.def("initTextures", [](Node* n)
+    simulation.def("glewInit", []()
```

Comment:
Oh, a new reviewer, sweet ! :-) 

`initTextures` here is independent of the back-end used (GL, Vulkan, ...), its binding available with or without OpenGL. `glewInit` however is only for GL, so it is activated only if SOFA has OpenGL activated.

Reply:
I agree that we need to remove the dependency to opengl...but this should be done with proper refactoring and not through #ifdef design. Of course this requires more work but this is for investment for a long term good :)

The initial code was there for experimental purpose, basically to see if rendering from a python rendering loop was possible. So  I have no problem if we break the API there to correctly split that part. Actually this could even lead to some cleaning in the core of sofa :)

Reply:
I agree, but this is already not to bad in SOFA (since the PR of @fredroy that isolated most of the GL in the drawToolGL. In SOFA, the choice of the backend then made in the viewer.

Here we don't have a viewer, I guess we could bind the `VisualParam` and `drawToolGl` and pass it to the `simulation::draw` somehow.

I propose we merge this fix and create an issue, as I have no time to do this right now. This is blocking for us as we need to run scripts on headless server nodes without GL.

Reply:
No way to merge #ifdef because this will delegate to someone else fixing the code one day. 
But make a PR where you simply remove the feature (as it shouldn't have been made) and I will merge it  :)

Reply:
Hey @damienmarchal 
I changed this PR instead of creating a new one, it should be good now